pkg/juicefs/mount/builder/vci-serverless.go

```diff
@@ -64,13 +71,27 @@ func NewVCIBuilder(setting *config.JfsSetting, capacity int64, app corev1.Pod, p
 func (r *VCIBuilder) NewMountSidecar() *corev1.Pod {
```

Comment:
if not too much trouble, could you apply the changes to all `pkg/juicefs/mount/builder/*serverless.go` codes? I imagine the code will be all similar?

this way, if tests pass for `pkg/juicefs/mount/builder/serverless.go`, we're quite confident that it's going to work for other platforms as well?

Reply:
only vci requires this annotation

Reply:
*serverless.go have a lot of similar code, which can be optimized later.